vspace/src/main/java/edu/asu/diging/vspace/core/data/display/SpaceLinkDisplayRepository.java

```diff
@@ -20,6 +20,9 @@
     @Query("SELECT d FROM SpaceLinkDisplay d WHERE d.link.sourceSpace.id = ?1")
     public List<SpaceLinkDisplay> findSpaceLinkDisplaysForSpace(String spaceId);
 
+    @Query("SELECT d FROM SpaceLinkDisplay d WHERE d.link.sourceSpace.id = ?1 AND d.link.targetSpace.spaceStatus = ?2")
+    public List<SpaceLinkDisplay> findSpaceLinkDisplaysForPublishedSpace(String spaceId, SpaceStatus spaceStatus);
```

Comment:
bad method name, does it get links to published spaces or links for a published space? but why does it get an extra space status? (I know what the method is supposed to do, but these would be the questions I'd ask if I wouldn't).